deployment/ansible/roles/driver.app/tasks/firewall.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Enable firewall
+  ufw: state=enabled policy=reject
+
+- name: Allow ssh access
```

Comment:
Yeah, I considered that as well. It's a bit of a toss up. I think I'd prefer to keep them separated out so they're more explicit (plus the nice output), and individual items can potentially be modified more easily if needed. But it definitely does add some repetition.